docs/taxii.md

```diff
@@ -0,0 +1,35 @@
+| [Home](../README.md) |
+|----------------------|
+
+# TAXII Server Integration

```

Comment:
Screenshots are missing with example is required

Reply:
Added.